owlbot.py

```diff
@@ -13,41 +13,80 @@
 # limitations under the License.
 
 """This script is used to synthesize generated parts of this library."""
+from pathlib import Path
+from typing import List, Optional
+
 import synthtool as s
 from synthtool import gcp
 from synthtool.languages import python
 
 common = gcp.CommonTemplates()
 
+# This is a customized version of the s.get_staging_dirs() function from synthtool to
+# cater for copying 2 different folders from googleapis-gen
+# which are datastore and datastore/admin
```

Comment:
[spanner](https://github.com/googleapis/python-spanner/tree/main/google/cloud) and [firestore](https://github.com/googleapis/python-firestore/tree/main/google/cloud), [bigtable](https://github.com/googleapis/python-bigtable/tree/main/google/cloud) all have a similar layout with a main client and at least one other client. Is it worth generalizing this function in `synthtool` instead?

Reply:
I opened a PR in the synthtool repo to propose this change but I was not able to get it approved. https://github.com/googleapis/synthtool/pull/1107